workflow/rules/common.smk

```diff
@@ -59,3 +74,27 @@ def get_input_for_mapfile(wildcards):
     doneFiles = expand(config['output'] + "{{Organism}}/{{refGenome}}/" + config['gvcfDir'] + "{sample}/" + "L{{list}}.done", sample=sample_names, **wildcards)
     print({'gvcfs': gvcfs, 'gvcfs_idx': gvcfs_idx, 'doneFiles': doneFiles})
     return {'gvcfs': gvcfs, 'gvcfs_idx': gvcfs_idx, 'doneFiles': doneFiles}
+
```

Comment:
Totally agree with this. Ran into this with Hifi assemblies. I will work on implementing this, though I believe the logic should live in the interval creating function?

Reply:
Yes, logic should definitely live in the interval creation function. This would we should definitely fix before closing the pull request, thanks for taking a look.